src/Prjkt/Component/Repofuck/Repofuck.php

```diff
@@ -307,13 +307,17 @@ protected function map(array $inserts, array $keys = [], $entity = null) : Model
 			throw new EntityNotDefined;
 		}
 
+		$fillable = $entity->getFillable();
+		
 		foreach($inserts as $key => $val)
 		{
 			if ( count($keys) > 0 && ! in_array($keys, $key) ) {
 				break;
 			}
 
-			$entity->{$key} = $val;
+			if (array_key_exists($key, $fillable)){
```

Comment:
This procedure is invalid. As there is validation in L314~L316. This should only run with the following constraints,

* `$keys` is empty.
* `$entity` has no attributes through `getAttributes` function